### PR TITLE
Support browser_web tests

### PR DESCRIPTION
**Pull request by FedericoCeratto**

### CODE REVIEW

File: fastpath/fastpath/core.py [hunk 1]
```diff
@@ -1449,6 +1449,25 @@ def score_openvpn(msm: dict) -> dict:
...
+    res = g_or(tk, "result", "")
+    if res == "ok":
+        pass
+    elif res == "error":
+        scores["blocking_general"] = 1.0
+    else:
+        scores["accuracy"] = 0.0
+
+    bn = g_or(tk, "browser_name", "")
+    scores["extra"] = dict(browser_name=bn)
```
Comment (hellais) [inline fastpath/fastpath/core.py:1465]: Can you add `load_time_ms` in here as well?

**Review by hellais — APPROVED** [review]
LGTM

**Comment by FedericoCeratto** [comment]
0.72~pr415-246 deployed on test

**Comment by FedericoCeratto** [comment]
deployed on FSN

**Pull request merged**